Description: Changed hard-coded xterm into x-terminal-emulator per Policy
 .
 In keeping with Debian policy, change the option --title to -T,
 and document the Debian specific change.
 .
 Converted with metadata by Vincent W. Chen <vinmun@gmail.com>.
 .
 Updated by Jaimos Skriletz <jaimosskriletz@gmail.com> to apply this
 change in the default-config and fvwm-menu-desktop.
Author: Manoj Srivastava <srivasta@debian.org>
Forwarded: not-needed
Last-Update: 2026-02-12
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/default-config/config
+++ b/default-config/config
@@ -29,7 +29,7 @@
 #
 # The following is used in various menus and also sets the terminal
 # that FvwmConsole uses. Change this to your terminal of choice
-InfoStoreAdd terminal xterm
+InfoStoreAdd terminal x-terminal-emulator
 
 # This is used for "Run Command" and the Meta+Space key-binding.
 # runcmd is the binary name and runcmdopt contains additional
--- a/modules/FvwmForm/FvwmForm-XDGMenuHelp
+++ b/modules/FvwmForm/FvwmForm-XDGMenuHelp
@@ -66,7 +66,7 @@ DestroyModuleConfig  FvwmForm-XDGMenuHel
 *FvwmForm-XDGMenuHelp:   Button  continue "$[gt.Options Help]"
 *FvwmForm-XDGMenuHelp:   Command Module FvwmForm FvwmForm-XDGOptionsHelp
 *FvwmForm-XDGMenuHelp:   Button  continue "fvwm-menu-desktop $[gt.manpage]"
-*FvwmForm-XDGMenuHelp:   Command Exec exec xterm -g 100x50 -n "$[gt.Help] fvwm-menu-desktop" -T "[$gt.Help] fvwm-menu-desktop" -e "man fvwm-menu-desktop"
+*FvwmForm-XDGMenuHelp:   Command Exec exec x-terminal-emulator -g 100x50 -n "$[gt.Help] fvwm-menu-desktop" -T "[$gt.Help] fvwm-menu-desktop" -e "man fvwm-menu-desktop"
 *FvwmForm-XDGMenuHelp:   Line    Left
 *FvwmForm-XDGMenuHelp:   Line    " "
 
--- a/modules/FvwmForm/FvwmForm-XDGOptionsHelp
+++ b/modules/FvwmForm/FvwmForm-XDGOptionsHelp
@@ -134,7 +134,7 @@ DestroyModuleConfig  FvwmForm-XDGOptions
 *FvwmForm-XDGOptionsHelp:   Button  continue "$[gt.XDGMenu Help]"
 *FvwmForm-XDGOptionsHelp:   Command Module FvwmForm FvwmForm-XDGMenuHelp
 *FvwmForm-XDGOptionsHelp:   Button  continue "fvwm-menu-desktop $[gt.manpage]"
-*FvwmForm-XDGOptionsHelp:   Command Exec exec xterm -g 100x50 -n "$[gt.Help] fvwm-menu-desktop" -T "$[gt.Help] fvwm-menu-desktop" -e "man fvwm-menu-desktop"
+*FvwmForm-XDGOptionsHelp:   Command Exec exec x-terminal-emulator -g 100x50 -n "$[gt.Help] fvwm-menu-desktop" -T "$[gt.Help] fvwm-menu-desktop" -e "man fvwm-menu-desktop"
 *FvwmForm-XDGOptionsHelp:   Line    Left
 *FvwmForm-XDGOptionsHelp:   Line    " "
 
--- a/bin/fvwm-menu-desktop.in
+++ b/bin/fvwm-menu-desktop.in
@@ -274,7 +274,7 @@ Standard output is a series Fvwm command
     regen_cmd = 'PipeRead `fvwm-menu-desktop`'
     dynamic_menu = False
     menu_error = False
-    term_cmd = "xterm -e"
+    term_cmd = "x-terminal-emulator -e" # Debian Policy change xterm to x-terminal-emulator
 
     # Loads config options from $FVWM_USERDIR/.FvwmForm-XDGMenu-Config
     if "FVWM_USERDIR" in os.environ:
@@ -888,7 +888,7 @@ Options:
     --regen-cmd ACTION        The fvwm ACTION for the 'Regenerate' menu item.
                               Default: 'PipeRead `fvwm-menu-desktop`'
     --term-cmd CMD            Terminal emulator CMD used on terminal entries.
-                              Default: xterm -e
+                              Debian Default: x-terminal-emulator -e
     --dynamic                 used with dynamic menus.
     -e, --menu-error          out python-xdg not found error in menu.
     -v, --verbose             run and display debug info on STDERR."""
--- a/doc/fvwm-menu-desktop.adoc
+++ b/doc/fvwm-menu-desktop.adoc
@@ -88,6 +88,10 @@ script exits with an error.
   to run terminal applications in. _CMD_ needs to end with an execute
   option, such as xterm's -e option, which will run the command that is
   appended to _CMD_. The default is "xterm -e".
++
+Debian specific: Per Debian policy, default changed to
+"x-terminal-emulator -e".
++
 *--dynamic*::
   This option is to be used with creating dynamic menus using
   *DynamicPopUpAction* and *DynamicPopDownAction*. This adds a
